core/tests/ts-integration/tests/l2-erc20.test.ts

```diff
@@ -0,0 +1,338 @@
+/**
+ * This suite contains tests checking default ERC-20 contract behavior.
+ */
+
+import { TestMaster } from '../src';
+import { Token } from '../src/types';
+import { shouldChangeTokenBalances, shouldOnlyTakeFee } from '../src/modifiers/balance-checker';
+
+import * as zksync from 'zksync-ethers';
+import * as ethers from 'ethers';
+import { Provider, Wallet } from 'ethers';
+import { scaledGasPrice, waitUntilBlockFinalized, deployContract, readContract } from '../src/helpers';
+import { L2_DEFAULT_ETH_PER_ACCOUNT } from '../src/context-owner';
+import { sleep } from 'zksync-ethers/build/utils';
+
+// import ZkSyncERC20 from '../../../etc/ERC20/artifacts-zk/contracts/ZkSyncERC20.sol/ZkSyncERC20.json';
+
+describe('L2 native ERC20 contract checks', () => {
+    let testMaster: TestMaster;
+    let alice: zksync.Wallet;
+    let bob: zksync.Wallet;
+    let isETHBasedChain: boolean;
+    let baseTokenAddress: string;
+    let zkTokenAssetId: string;
+    let tokenDetails: Token;
+    let aliceErc20: zksync.Contract;
+    let l1NativeTokenVault: ethers.Contract;
+    let l1Wallet: Wallet;
+    let l2Wallet: Wallet;
+    let l1Provider: Provider;
+    let l2Provider: Provider;
+
+    beforeAll(async () => {
+        testMaster = TestMaster.getInstance(__filename);
+        alice = testMaster.mainAccount();
+        bob = testMaster.newEmptyAccount();
+        const bridgeContracts = await alice.getL1BridgeContracts();
+        const assetRouter = bridgeContracts.shared;
+        l2Provider = alice._providerL2();
+        l1Provider = alice._providerL1();
+        l2Wallet = new Wallet(alice.privateKey, l2Provider);
+        l1Wallet = new Wallet(alice.privateKey, l1Provider);
+        const L2_NATIVE_TOKEN_VAULT_ADDRESS = '0x0000000000000000000000000000000000010004';
+        const l2NtvInterface = readContract(
+            '../../../contracts/l1-contracts/artifacts/contracts/bridge/ntv',
+            'L2NativeTokenVault'
+        ).abi;
+        const l2NativeTokenVault = new ethers.Contract(L2_NATIVE_TOKEN_VAULT_ADDRESS, l2NtvInterface, l2Wallet);
+        const l1AssetRouterInterface = readContract(
+            '../../../contracts/l1-contracts/artifacts/contracts/bridge/asset-router',
+            'L1AssetRouter'
+        ).abi;
+        const l1NativeTokenVaultInterface = readContract(
+            '../../../contracts/l1-contracts/artifacts/contracts/bridge/ntv',
```

Comment:
better to remove the repetition in the path `../../../contracts/l1-contracts/artifacts`